src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfoOfT.cs

```diff
@@ -89,7 +89,7 @@ internal sealed class JsonPropertyInfo<T> : JsonPropertyInfo
                     }
             }
 
-            GetPolicies(ignoreCondition, isReferenceType: default(T) == null);
+            GetPolicies(ignoreCondition, defaultValueIsNull: default(T) == null);
```

Comment:
Could it be better `DeclaredPropertyType.IsValueType`?

Reply:
Won't work for nullable structs. Will change to `Converter.CanBeNull`.